### PR TITLE
Expo pagination issues and pr screen

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes:  #1708

- Refactoring  of Issues and PRs Pages/Screen
- Added page titles to each repo screen

Screenshot 2023-03-30 at 12 52 57 PM

simulator_screenshot_E7D1AF6A-3922-47DF-A3A7-B60098B26C1C

### CODE REVIEW

File: expo-zustand-styled-components/src/components/PRAndIssueHeader/PRAndIssueHeader.tsx [hunk 1]
```diff
@@ -81,33 +72,33 @@ const PRAndIssueHeader = ({ cardType, openCount, closedCount }: PRAndIssueHeader
...
-          showOptions={showOptions}
           selected={label}
           items={labelOptions()}
-          itemsColors={labelOptionsColors()}
+          showOptions={showOptions}
           selectOption={selectLabel}
-          setShowOptions={(value) => setShowOptions(value)}
           style={filterDropdownStyle}
+          itemsColors={labelOptionsColors()}
+          setShowOptions={(value) => setShowOptions(value)}
```
[the remark this answers is not in the document]
Comment (AllStackDev1) [reply expo-zustand-styled-components/src/components/PRAndIssueHeader/PRAndIssueHeader.tsx:81]: lol for beauty

File: expo-zustand-styled-components/src/hooks/stores/useIssuesStore.ts [hunk 2]
```diff
@@ -18,53 +18,20 @@ interface Issues {
...
-  issues: Issues;
-  setLoading: (value: boolean) => void;
-  setIssues: (value: Issues) => void;
-  setBefore: (value: string) => void;
-  setAfter: (value: string) => void;
-  setErrorMsg: (value: string) => void;
-  resetBeforeAndAfter: () => void;
+  issues?: Issues;
+  isLoading: boolean;
+  _issues: Map<string, Issues>;
```
Comment (vyktoremario) [inline expo-zustand-styled-components/src/hooks/stores/useIssuesStore.ts:24]: I am a bit confused about this. Why do we have this property and why is it a map and how is is different from `issues?: Issues`?

File: expo-zustand-styled-components/src/hooks/stores/usePullRequestsStore.ts [hunk 3]
```diff
@@ -18,52 +18,19 @@ interface PullRequests {
...
-  pullRequests: PullRequests;
-  setLoading: (value: boolean) => void;
-  setPullRequests: (value: PullRequests) => void;
-  setBefore: (value: string) => void;
-  setAfter: (value: string) => void;
-  setErrorMsg: (value: string) => void;
-  resetBeforeAndAfter: () => void;
+  isLoading: boolean;
+  pullRequests?: PullRequests;
+  _pullRequests: Map<string, PullRequests>;
```
Comment (vyktoremario) [inline expo-zustand-styled-components/src/hooks/stores/usePullRequestsStore.ts:24]: Same comment as above

**Review by vyktoremario — COMMENTED** [review]
I just have questions

File: expo-zustand-styled-components/src/hooks/stores/useIssuesStore.ts (hunk 2 above)
Comment (AllStackDev1) [reply expo-zustand-styled-components/src/hooks/stores/useIssuesStore.ts:24]: temporal caching

File: expo-zustand-styled-components/src/hooks/stores/usePullRequestsStore.ts (hunk 3 above)
Comment (AllStackDev1) [reply expo-zustand-styled-components/src/hooks/stores/usePullRequestsStore.ts:24]: temporal caching

**Review by markshenouda — APPROVED** [review]
Looks good 👍

**Pull request merged**